digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -647,6 +647,62 @@ object DataFixture {
     println("\n")
   }
 
+  def listingBusStopsWithSideCodeConflictWithRoadLinkDirection(): Unit = {
+    println("\nCreate a listing of bus stops on one-way roads in Production that have side code against traffic direction of road link")
+    println(DateTime.now())
+
+    var persistedStop: Seq[PersistedMassTransitStop] = Seq()
+    var conflictedBusStopsOTH: Seq[PersistedMassTransitStop] = Seq()
+
+    //Get All Municipalities
+    val municipalities: Seq[Int] =
+      OracleDatabase.withDynSession {
+        Queries.getMunicipalities
+      }
+
+    println("Bus stops with side code in conflict By Municipality")
+
+    municipalities.foreach { municipality =>
+      println("Start processing municipality %d".format(municipality))
+
+      //Get all OTH Bus Stops By Municipality
+      persistedStop = massTransitStopService.getByMunicipality(municipality, false)
+
+      persistedStop.foreach { stop =>
+        val massTransitStopDirectionValue = stop.validityDirection
+
+        val roadLinkOfMassTransitStop = roadLinkService.getRoadLinkFromVVH(stop.linkId)
```

Comment:
Could be faster to get all vvh road links by link ids at once: less overhead for HTTP connections, but speedup might be negligible and for a batch job this is good enough.